### DISCUSSION TITLE
How to update v3.1.0 to 3.2.0

### DISCUSSION THREAD

**Discussion by vsavchenko21** [chartbrew/chartbrew#214] [Q&A]
Is there some Docs/Guide how to update v3.1.0 to 3.2.0 on Server?

**Comment by razvanilin** [comment]
Hey @vsavchenko21 there's no guide currently, but there are two main ways that this can be done:

1. If using Docker: `docker pull razvanilin/chartbrew`
2. If using a Git fork, sync your fork from the original repository, then `git pull origin master`
3. If you cloned this repo directly, `git pull origin master`
